src/i18n.ts

```diff
@@ -102,6 +102,9 @@ export function translate(key: string, ...args: (string | number | boolean | nul
     return text;
 }
 
+/** Alias shortcut for translate function */
+export const t = translate;
```

Comment:
should that be also added to the readme?

Reply:
I could mention, that I always write I18n.t and not I18n.translate. It is not important. No one uses I18n in backend except me

Reply:
> No one uses I18n in backend except me

I am not into the context. However, that's not a good argument for writing sustainable code^^